### PR TITLE
Use ConnectFour pallet in SGX runtime

### PR DESCRIPTION
**Pull request by gaudenzkessler**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -284,6 +314,9 @@ construct_runtime!(
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
 		Parentchain: pallet_parentchain::{Pallet, Call, Storage},
+		RandomnessCollectiveFlip: pallet_randomness_collective_flip,
```
Comment (haerdib) [inline runtime/src/lib.rs:317]: Why are you not using the Pallet & Storage of this pallet? (https://github.com/ajuna-network/ajuna-node/blob/master/runtime/src/lib.rs#L288)

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -284,6 +314,9 @@ construct_runtime!(
...
+		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>},
+		ConnectFour: pallet_connectfour::{Pallet, Call, Storage, Event<T>},
```
[the remark this answers is not in the document]
Comment (gaudenzkessler) [reply runtime/src/lib.rs]: thank you. added

**Review by haerdib — APPROVED** [review]
LGTM, thanks 👍

**Comment by brenzi** [comment]
@gaudenzkessler I actually think Ajuna should fork this repo instead of maintaining a project-specific branch in here.

**Comment by haerdib** [comment]
Oh. Good catch. Ajuna already forked it: https://github.com/ajuna-network/sgx-runtime/tree/ajuna

**Pull request closed without merging**